src/libraries/Common/src/Interop/Windows/Crypt32/Interop.CertNameToStr.cs

```diff
@@ -7,12 +7,22 @@ internal static partial class Interop
 {
     internal static partial class Crypt32
     {
+#if DLLIMPORTGENERATOR_ENABLED
+        [GeneratedDllImport(Libraries.Crypt32, CharSet = CharSet.Unicode, SetLastError = true, EntryPoint = "CertNameToStrW")]
+        internal static unsafe partial int CertNameToStr(
+            int dwCertEncodingType,
+            void* pName,
+            int dwStrType,
+            char* psz,
+            int csz);
+#else
         [DllImport(Libraries.Crypt32, CharSet = CharSet.Unicode, SetLastError = true, EntryPoint = "CertNameToStrW")]
         internal static extern unsafe int CertNameToStr(
             int dwCertEncodingType,
             void* pName,
             int dwStrType,
             char* psz,
             int csz);
+#endif
```

Comment:
For all of these where it applies, consider putting the if/else around just the part that differs? e.g.
```C#
#if DLLIMPORTGENERATOR_ENABLED
        [GeneratedDllImport(Libraries.Crypt32, CharSet = CharSet.Unicode, SetLastError = true, EntryPoint = "CertNameToStrW")]
        internal static unsafe partial
#else
        [DllImport(Libraries.Crypt32, CharSet = CharSet.Unicode, SetLastError = true, EntryPoint = "CertNameToStrW")]
        internal static extern unsafe
#endif
        int CertNameToStr(
            int dwCertEncodingType,
            void* pName,
            int dwStrType,
            char* psz,
            int csz);
```
?